Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.ComboBox/Controls Toolkit/KryptonComboBox.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System;
```

Comment:
This class should be removed as it does not appear to do "Anything" that the normal KCB does .
In fact it looks like it does less

Reply:
I think this one might be a little different, as upon testing, I found that the item hover colour was setup differently (gradients).

Reply:
Then that needs to be in the name, Or add gradients to the base toolkit

Reply:
> Then that needs to be in the name, Or add gradients to the base toolkit

Good point, new feature request?

Reply:
Does not need a new Feature as it already exists in the Toolkit as part of the tracking state:
![image](https://user-images.githubusercontent.com/2418812/119253667-36040680-bbaa-11eb-9a93-9cf8aabb31c0.png)

Via:
![image](https://user-images.githubusercontent.com/2418812/119253680-44522280-bbaa-11eb-9950-829fb07ef41a.png)

Reply:
I've taken them out, **but** left the 'tree' one in.